react-app/src/components/googleOAuth/googleOAuth.js

```diff
@@ -0,0 +1,44 @@
+import GoogleLogin from "react-google-login";
```

Comment:
fixed the casing for all components in other branch. E.g googleOAuth -> GoogleOAuth